src/components/BookComponent/EACHBOOK.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import style from './eachBook.module.css';
+import { RemoveBook } from '../../redux/books/books';
+import Percentage from './myPercentage';
+
+function EachBookEach(props) {
+  const {
+    category, title, author, id,
+  } = props;
+  const dispatchBook = useDispatch();
+  return (
+    <div className={style.leftside}>
+      <div className={style.ulBody}>
+        <ul>
+          <li className={style.category}>{category}</li>
+          <li className={style.title}>{title}</li>
+          <li className={style.author}>{author}</li>
+        </ul>
+        <div className={style.button}>
+          <button type="button">Comments</button>
+          |
+          <button type="button" onClick={() => dispatchBook(RemoveBook(id))}>Remove</button>
+          |
+          <button type="button">Edit</button>
+        </div>
```

Comment:
- [ ] I will also suggest that you ensure that each book card stands alone as seen in the design and is well spaced as seen in the original design:wine_glass: 
- Your Design
![image](https://github.com/citec-47/bookstoreReact/assets/102699958/adb208bd-fb04-4310-b3c8-2c49cc54e5dd)

- The Zeplin Design
![image](https://github.com/citec-47/bookstoreReact/assets/102699958/bd24a03a-762c-44f3-b57e-cadbde71db75)